Cannot find module 'nx/bin/nx' when running create-nx-workspace@latest (works with 16.8.0)

### Current Behavior

### Current Behavior

Running:
npx -y create-nx-workspace@latest jobflow-monorepo --preset=apps --nxCloud=skip --skip-install --skip-prettier --verbose

Fails with:
Error: Cannot find module 'nx/bin/nx'
(see full stack trace below)




### Expected Behavior

### Expected Behavior

Workspace should be created successfully (as it does with 16.8.0).

### GitHub Repo

_No response_

### Steps to Reproduce

### Steps to Reproduce

1. npm uninstall -g nx @nrwl/cli @nrwl/nx
2. npm install -g nx
3. npm cache clean --force
4. rmdir /s /q jobflow-monorepo
5. npx -y create-nx-workspace@latest jobflow-monorepo --preset=apps --nxCloud=skip --skip-install --skip-prettier --verbose

### Environment

- Windows 10
- Node.js vXX.XX.XX
- npm vX.X.X

### Additional Info

- Downgrading to 16.8.0 works.
- Tried all cache clean, uninstall, and folder deletion steps.

### Stack Trace

(paste the full error log here)


### Nx Report

```shell
Node   : 18.20.8
OS     : win32-x64
npm    : 10.9.3
nx             : 16.8.0
@nx/js         : 16.8.0
@nx/workspace  : 16.8.0
@nx/devkit     : 16.8.0
@nrwl/tao      : 16.8.0
```

### Failure Logs

```shell
NX   Cannot find module 'nx/bin/nx'

Require stack:
- C:\Users\SHINDA\AppData\Local\Temp\tmp-11336-jickiTlDTt0r\node_modules\@nx\workspace\src\generators\new\generate-preset.js
- C:\Users\SHINDA\AppData\Local\Temp\tmp-11336-jickiTlDTt0r\node_modules\@nx\workspace\src\generators\new\new.js
- C:\Users\SHINDA\AppData\Local\Temp\tmp-11336-jickiTlDTt0r\node_modules\nx\src\config\schema-utils.js
- C:\Users\SHINDA\AppData\Local\Temp\tmp-11336-jickiTlDTt0r\node_modules\nx\src\command-line\run\executor-utils.js
- C:\Users\SHINDA\AppData\Local\Temp\tmp-11336-jickiTlDTt0r\node_modules\nx\src\project-graph\utils\project-configuration-utils.js
- C:\Users\SHINDA\AppData\Local\Temp\tmp-11336-jickiTlDTt0r\node_modules\nx\src\utils\package-json.js
- C:\Users\SHINDA\AppData\Local\Temp\tmp-11336-jickiTlDTt0r\node_modules\nx\bin\nx.js
Error: Cannot find module 'nx/bin/nx'
```

### Package Manager Version

10.9.3

### Operating System

- [ ] macOS
- [ ] Linux
- [x] Windows
- [ ] Other (Please specify)

### Additional Information

Additional Information
The issue only occurs with the latest Nx CLI (@latest), but works with version 16.8.0.
All cache, uninstall, and folder deletion steps were tried.
The error occurs even in a completely empty directory.
The command used: npx -y create-nx-workspace@latest nameofmyrepo-monorepo --preset=apps --nxCloud=skip --skip-install --skip-prettier --verbose
The error is reproducible on Windows 10, Node.js 18.20.8, npm 10.9.3.

--- a/packages/create-nx-workspace/src/create-empty-workspace.ts
+++ b/packages/create-nx-workspace/src/create-empty-workspace.ts
@@ -33,6 +33,11 @@ export async function createEmptyWorkspace<T extends CreateWorkspaceOptions>(
 
   const directory = options.name;
 
+  // Cannot skip install for create-nx-workspace or else it'll fail.
+  // Even though --skipInstall is not an option to create-nx-workspace, we pass through extra options to presets.
+  // See: https://github.com/nrwl/nx/issues/31834
+  delete (options as any).skipInstall;
+
   const args = unparse({
     ...options,
   }).join(' ');

